ChangeLog

```diff
@@ -1,3 +1,69 @@
+* Tue Apr 28 2020 Klaus Wenninger <klaus.wenninger@aon.at> Pacemaker-2.0.4
+- Changesets: 442
+- Diff: 269 files changed, 21845 insertions(+), 13363 deletions(-)
+
+- Features added since Pacemaker-2.0.3
+  + build: Add support for glib-based unit tests.
+  + controller: add new feature 'shutdown-locks'
+        Optionally allow a gracefully shutdown node to have the resources
+        locked to it for a configurable time.
+        So that it can be restarted with exactly the same resources
+        running as before.
+  + controller/fencer/scheduler: add new feature 'priority-fencing-delay'
+        Optionally derive the priority of a node from the resource-priorities
+        of the resources it is running.
+        In a fencing-race the node with the highest priority has a certain
+        advantage over the others as fencing requests for that node are
+        executed with an additional delay.
+        controlled via cluster option priority-fencing-delay (default = 0)
+  + stonith_admin: add --delay option (default = 0) to support enforced
+                   fencing delay
+  + tools: Add --include= and --exclude= to crm_mon.
+  + tools: Add --node= to filter crm_mon output for a node (or tag).
+  + tools: Add --failures for printing failed resource actions to crm_mon.
```

Comment:
This is actually a "section" for --include/--exclude ... and since those are new this release, we can drop this